backend/Dockerfile

```diff
@@ -1,19 +1,17 @@
-FROM strapi/base:alpine
-
-ARG STRAPI_VERSION
-RUN yarn global add @strapi/strapi@${STRAPI_VERSION}
-
-WORKDIR /srv/app
-
-COPY ./package*.json ./
-RUN npm install
-
-COPY ./src/ ./src/
-COPY ./config/ ./config/
-COPY ./database/ ./database/
-COPY ./public/ ./public/
-COPY ./favicon.ico ./
-
-RUN npm run build
-
-CMD ["npm", "start"]
\ No newline at end of file
+FROM node:16-alpine
+# Installing libvips for sharp compatibility
+RUN apk --update-cache add vips-dev \
+ && rm -rf /var/cache/apk/*
+ARG NODE_ENV=production
+ENV NODE_ENV=${NODE_ENV}
+WORKDIR /opt/
+COPY ./package.json ./
+COPY ./yarn.lock ./
+ENV PATH /opt/node_modules/.bin:$PATH
+RUN yarn config set network-timeout 600000 -g
```

Comment:
That seems pretty large. Is this just a testing line, or should it stay in here?

Reply:
@andyg0808 
This Dockerfile basically comes from [this article](https://blog.dehlin.dev/docker-with-strapi-v4) which is the community article currently recommended by the Strapi team.

This particular line seems to be there to make Yarn more resilient on slow networks:
 
<img width="790" alt="Screenshot 2022-05-02 at 10 45 18" src="https://user-images.githubusercontent.com/4640135/166208300-0a08f7a8-8112-4620-b8b3-05b426afcf3d.png">

Not sure if we need it, but I don't see any harm in keep it right?

Reply:
If the worst thing about this is that it's just an unconventional timeout length and we can afford to leave this be without incurring any real performance penalty, then I probably wouldn't worry about changing it for now.

Reply:
Yup, totally on board with that.